src/olympia/files/models.py

```diff
@@ -589,7 +592,7 @@ def save(self, *args, **kw):
     def add_file(self, chunks, filename, size):
         if not self.uuid:
             self.uuid = self._meta.get_field('uuid')._create_uuid().hex
-        filename = smart_str(u'{0}_{1}'.format(self.uuid, filename))
+        filename = force_bytes(u'{0}_{1}'.format(self.uuid, filename))
```

Comment:
This seems like a backwards-incompatible change: with the old field, `self.uuid` was a hyphen-less 32 chars string. Here you'll end up with the 36 chars w/ hyphen variant.